app/src/main/java/net/squanchy/injection/ContextModule.java

```diff
@@ -0,0 +1,21 @@
+package net.squanchy.injection;
+
+import android.content.Context;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public class ContextModule {
```

Comment:
Well if it's needed in the `ApplicationComponent` then it _must_ be the `Application`. Change this to be `ApplicationContextModule` and accept and provide an `Application` and we are all set.